server/app/views/components/FieldWithLabel.java

```diff
@@ -87,6 +88,11 @@ public class FieldWithLabel {
   private boolean required = false;
   private boolean ariaRequired = false;
   private boolean focusOnInput = false;
+  private boolean markdownSupported = false;
+  private String markdownText = "";
+  private String markdownLinkText = "";
+  private String markdownLinkHref =
```

Comment:
(suggestion, non-blocking) Since you won't show these if `markdownSupported` is false, what do you think about setting them to a default value. That way you only have to call `setMarkdownSupported(true)` in most cases and only override the markdown text in the one spot that you have "some markdown is supported.

Reply:
Good idea!

Reply:
I can't set a default for the link text since we rely on the `isBlank` check to render the link, but I set `markdownText` to a default which at least cleans up the program details use case.